actor-tests/src/test/scala/com/evolutiongaming/akkaeffect/ActorOfTest.scala

```diff
@@ -168,35 +164,32 @@ class ActorOfTest extends AsyncFunSuite with ActorSuite with Matchers {
     } yield result
   }
 
-
   private def receive[F[_]: Async: ToFuture: FromFuture](
     actorSystem: ActorSystem,
-    shift: F[Unit]
+    shift: F[Unit],
   ): F[Unit] = {
 
     case class GetAndInc(delay: F[Unit])
 
     def receiveOf = ReceiveOf.const {
       val receive = for {
         state <- Ref[F].of(0)
-      } yield {
-        Receive[Call[F, Any, Any]] { call =>
-          call.msg match {
-            case a: GetAndInc =>
-              for {
-                _ <- shift
-                _ <- a.delay
-                a <- state.modify { a => (a + 1, a) }
-                _ <- call.reply(a)
-              } yield false
-
-            case _ => shift as false
-          }
-        } {
-          false.pure[F]
+      } yield Receive[Call[F, Any, Any]] { call =>
+        call.msg match {
+          case a: GetAndInc =>
+            for {
+              _ <- shift
+              _ <- a.delay
+              a <- state.modify(a => (a + 1, a))
+              _ <- call.reply(a)
+            } yield false
+
+          case _ => shift as false
         }
+      } {
+        false.pure[F]
       }
-      Resource.make { shift productR receive } { _ => shift }
+      Resource.make(shift productR receive)(_ => shift)
```

Comment:
IMO using `{` and `}` makes code more readable, but **its purely personal preference** thus please ignore if you disagree 👍

Reply:
this is how scalafmt deals with one-liners and that config was there already 🤷 

I didn't explore scalafmt configuration page for option to keep curly braces for one-liners, but it might be that following code would keep curly braces:
```scala
Resource.make { 
  shift productR receive
} { _ => 
  shift
}
```